### ISSUE TITLE
[Epic] "Downloadable reports" domain from mhv-medical-records

### ISSUE THREAD

**Issue by jonathanjnelson** [department-of-veterans-affairs/va.gov-team#53881]
## What does your team need support for? Check all that apply.
- [x] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Something else 
 
 *Give a brief description of what your team needs support with.*

We will need content help to really make clear to users what _VA health summary_ is, _VA Blue Button_ is, and what the difference between the two are. 

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [x] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?

  TBD, but the current estimate is 2024

*Please provide an estimated date so our team can create other relevant tickets.*
  
## Supporting artifacts
 - Link to product outline: [Medical Records Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records)
 - [Link to Downloadable Reports: Domain Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/data-domains/downloadable-reports/downloadable-reports.md)
 - [Link to prototype or mockups](https://www.sketch.com/s/0803544d-e78c-423a-a70e-7db01a9a31c7/p/0816C226-038F-418F-A2CB-883E3951FA6F/canvas) 
 - [Blue button Report PDF (Hi-Fi mocks)](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/p/948A0CC5-6470-4732-83AB-EEEE67C9EFAB/canvas#Version)
 - [Independent Vaccine record downloadable PDF](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/33E09B2D-A68E-44C6-8059-30EBD4A70580)
 - [Vaccine record list downloadable PDF](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/C3C0B132-4959-4E96-8764-AC1D78711BD7)
      - The format of these two vaccine record pages will be followed for all of the domains. 

## Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: TBD - this is only one domain out of the larger medical records product.
- Estimated staging review date: TBD - this is only one domain out of the larger medical records product.
- Content and IA work needed by: We'd like to conduct usability testing with approved content, which should happen late February at the earliest.

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: mhv-medical-records
- OCTO-DE product owner: Lauren Alexanderson
- Product manager: Bryan Riley
- Designer: Alexia Wunder, Bobby Bailey 
- FE engineer: Mike Moyer
- Product/team Slack channel: #mhv-medical-records

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by laurwill** [comment]
Hi @jonathanjnelson @coultonbunney-usds , 
We have some questions about the data sources and use cases here. Apologies if these are answered elsewhere.

**Data sources**
The domain brief says "Data for the VA Health Summary report is derived from the Veterans Health Information Exchange (VHIE). Data for the Blue Button Report is derived from VistA, and also includes self-reported data." 
- Is the data shared in VHIE also originally derived from VistA (at least for non-Cerner patients)?
- Is the only difference between VA Health Summary and VA Blue Button the inclusion of self-entered data? Or do we have other specific differences documented somewhere?
- Does VHIE share full, complete VA medical records with non-VA providers? Does it share self-entered data from Blue Button?

**Use cases**
Domain brief lists "share health information with non-VA health care providers" as a common use case, and "take a digital copy of their health information to a health care provider on a thumb drive or other portable storage device" as an edge case. 
- How do we expect people to share downloadable reports with providers, other than using a thumb drive? Are we expecting them to bring printed records on paper, send them through secure messaging, or some other way?
- The file size limit on attachments in secure messaging is 6MB. Would the average VA Health Summary or VA Blue Button download be too large to send through SM?
- Are we concerned about users sending their downloaded reports via regular email (which would not protect their PHI)?

**Comment by laurwill** [comment]
Update: From a quick comparison of sample Blue Button and Health Summary reports, these seem to be the 2 key differences:
- Health summary doesn't include self-entered data
- Health summary only displays most recent data in each domain — for example, results from the last 15 months, 18 months, or 24 months

So the name of "Health summary" is misleading — it doesn't "summarize" what's in Blue Button, it shows all the same data for each record type. So you get all the same info, just fewer and more recent records of each type. 

Does this sound accurate?

**Comment by coultonbunney-usds** [comment]
> The domain brief says "Data for the VA Health Summary report is derived from the Veterans Health Information Exchange (VHIE). Data for the Blue Button Report is derived from VistA, and also includes self-reported data."

This might not be exactly correct. We spoke with OCC folks, and it turns out that if a patient opts out of sharing their info with VHIE, it does not impact their VA Health Summary. Therefore, it must be the case that the VA Health Summary is created and then shared with VHIE if you're opted in. I need to confirm this, but our working assumption should be that the Health Summary is not derived from VHIE, but rather it is shared with VHIE. It is actually derived from VistA. 

> Is the data shared in VHIE also originally derived from VistA (at least for non-Cerner patients)?

See above. Short answer, yes. 

> Is the only difference between VA Health Summary and VA Blue Button the inclusion of self-entered data? Or do we have other specific differences documented somewhere?

VA Health Summary does not include self-entered data. But there are other differences. 
* VA Health Summary follows the CCD format. Blue Button does not. 
* The CCD format dictates that VA Health Summary include specific information. Not all of this information is included as part of Blue Button. These differences include
  * VA Health Summary includes encounter notes, procedure notes, history and physical notes, and consultation notes. Blue Button only includes these note types if they have been categorized in VistA as progress notes or consult result notes. Most of the aforementioned notes would not normally be characterized as such.
  * VA Health Summary includes social history. BB does not. 
  * VA Health Summary includes Advanced directive. BB does not.
  * VA Health Summary includes Functional status. BB does not. 
  * VA Health Summary includes Plan of Care / Treatment. BB does not. 
 * Some of these categories are limited as part of VA Health Summary. 
   * You can find the the business rules here: https://www.va.gov/BLUEBUTTON/docs/MyHealtheVet_VA_Health_Summary_Data_Business_Rules_October2017.pdf
  
> Does VHIE share full, complete VA medical records with non-VA providers? Does it share self-entered data from Blue Button?

VHIE does not. I believe it shares the VA Health Summary. In our research, we heard that it may share an extended version of the summary, but we could not confirm.

**Comment by laurwill** [comment]
Hi @coultonbunney-usds ,
One consideration around including self-entered data in Blue Button:

If we don't include self-entered data in BB, I'm struggling to explain why a Veteran would choose to download BB instead of their Health Summary.

After closer comparison of the user guides and other documentation, and your answers above, it seems that:
- Both are time limited to ~2 years of recent records in the lab/test, vitals, and care notes/summaries domains (max 5 or 10 most recent)
- Both provide full allergy and vaccine lists
- Health summary provides additional records not in BB like certain types of summaries, procedure lists, advance directives, etc
- Health summary provides XML file type to upload into other systems

So if we don't include self-entered info in phase 0 and we can't say "download your BB if you want to get your self-entered info", what reason could we give for choosing to download BB?

**Comment by coultonbunney-usds** [comment]
Can you point me to where you found that BB only contains ~2 years of recent records in the lab/test, vitals, and care notes/summaries domains (max 5 or 10 most recent)?

My understanding is that BB allows a user to fetch data much further back in time, for example at least 10 years for notes.

**Comment by laurwill** [comment]
Posting an in-process draft here, to discuss in today's content sync: [https://app.zenhub.com/files/133843125/acc09a85-8f1f-4f62-97b5-4968ac30974b/download](https://app.zenhub.com/files/133843125/acc09a85-8f1f-4f62-97b5-4968ac30974b/download)

**Comment by laurwill** [comment]
@coultonbunney-usds I got that from the User Guide: [https://app.zenhub.com/files/133843125/ee5e8a9e-a199-4e6c-96e1-7203e93ddb95/download](https://app.zenhub.com/files/133843125/ee5e8a9e-a199-4e6c-96e1-7203e93ddb95/download)

But maybe that's incorrect! Apologies if I'm muddling things. 

So for phase 0, if we're not including a date range selection, BB would include records from the beginning of time?

**Comment by laurwill** [comment]
Hi @coultonbunney-usds , documenting updates from our chat in yesterday's design collab sync:
- BB has no self-reported data in phase 0
- We may or may not include a "download health summary" link in phase 0 (VHIE sharing only) — need to confirm this
- We don't want to spend too much time on the content to distinguish BB from Health Summary in case that ends up being irrelevant

I'm posting another in-process draft here that reflects the approach we discussed in syncs, in case it's helpful to use as an aid for discussions while I'm out. But noting that since this is a full page with more extensive content, we'll want to do another review and copyedit before it goes to testing. And any changes to the report features will trigger changes here.
[Share your medical records page content.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/11055022/Share.your.medical.records.page.content.docx)

Also noting that Ngozi confirmed BB data goes back farther than 2 years, and suggested using some Q&A sections on this page to explain the differences between BB and health summary — if we end up including download options for both, we can look at adding Q&A accordions or something similar.

cc @aliyahblackmore

**Comment by laurwill** [comment]
@alexiawunder @BobbyBaileyRB @coultonbunney-usds ,
Small updates to the Share your records page content are in the [Medical records working content draft](https://dvagov-my.sharepoint.com/:w:/g/personal/laura_willwerth_va_gov/Eacz1GNGqrdNn3b5gC1j0n4BjNyZ6LyafJCpEfebp-cPVg?e=f6e3Hy). 

I'll post content suggestions for the CBC PDF shortly.

cc @aliyahblackmore @AnneCostello

**Comment by laurwill** [comment]
Updates for the CBC PDF:
- The table approach for the results in the PDF was causing content issues (and most likely accessibility issues as well). It was difficult to fit the lab location and interpretation fields in the table in a way that made sense. So Bobby, Lexi, and I talked, and they've moved the content out of the table and into a format closer to the html version of CBC results. 
- I provided some working intro text for the top of the PDF (see below). For now, I've added a line about contacting your care team — we'll need to update this with more info about how to do so, but want to confirm best way to add a URL for MHV SM to a print material. It will likely be spelled out MHV homepage URL. We may also want a crisis line message here, as discussed on content sync. But it seems out of place in the CBC example, so holding off on adding this for the moment. I've noted these additional pieces in the content tracker.

**[H1] Lab and test results: Complete blood count on April 1, 2022**

If your results are outside the standard range, this doesn’t automatically mean you have a health problem. Your provider will explain what your results mean for your health. If you have questions, contact your VA care team.

**Comment by laurwill** [comment]
Hi @BobbyBaileyRB , I got most of the content for the downloadable PDFs into the content doc in Sharepoint. I'll update the rest by Monday morning!

**Comment by laurwill** [comment]
Hi again @BobbyBaileyRB , full suggested content for the downloadable PDFs is now in the [Medical records working content draft](https://dvagov-my.sharepoint.com/:w:/g/personal/laura_willwerth_va_gov/Eacz1GNGqrdNn3b5gC1j0n4BjNyZ6LyafJCpEfebp-cPVg?e=f6e3Hy). 

**Question:** Is deep-linking possible in these PDF materials? Or do all links need to go to MHV landing page? For now, I’ve left the links as TBD.

Here's a few notes:
- As we discussed in last week’s content sync, we’ll need to decide the best way to display URLs in PDFs. Options include spelling out all URLs in the PDF or leaving them as hyperlinks in the PDF and converting to spelled-out URLs when printing (if possible).
- I prioritized info from “What to know” boxes to shorten the intro content for print versions. For example, I only included the “self-entered info” line on full BB, not on other list PDFs.
- Tried out a brief crisis line message on test results and full BB.
- I didn’t include all field labels for each screen — we’ll need to update these to match recent updates to field labels in the html versions.
- After seeing more of these screens, suggest capping first letter of word after colon throughout. It looks inconsistent with lowercase words like “blood” between capped provider names and dates, etc. So we can diverge from AP style on this point.

Let me know if you have questions or want to chat about anything!

**Comment by laurwill** [comment]
Hi again @BobbyBaileyRB ,
As we chatted, I updated to make the content fit better in the PDF designs — 
- Trimmed content even more to keep the print products brief
- Made the crisis line language a footer element on single records/list records to avoid needing bullets at the top of these pdfs. (It will still be a bullet on the full BB report, not a footer element in full BB.)

Let me know if you want to talk as you're updating — happy to adjust content again if it's still causing issues in the designs.

And I think you said you would try out a couple options for displaying the link text — I've also called out some questions in the doc around links. For example, can we link back to a Veteran's allergy records from a PDF of their vaccine records? Or do we need to send them back to a landing page instead?
